docs/reference/time_series/api.md

```diff
@@ -0,0 +1,324 @@
+| content  | List\[TimeSeriesForecast] | Yes      | One or more time series forecasts (per input) |
+| error\*  | Error                     | No       | Error if this forecast failed                 |
+| ...extra | any                       | No       | Additional extensible fields                  |
+
+
+### TimeSeriesForecast
+
+| Field            | Type                   | Required | Description                                                |
+| ---------------- | ---------------------- | -------- | ---------------------------------------------------------- |
+| type             | TimeSeriesType         | Yes      | 'univariate\_time\_series' or 'multivariate\_time\_series' |
+| name             | string                 | Yes      | The name of the time series                                |
+| mean\_forecast   | TimeSeries             | Yes      | Mean (expected) forecast values                            |
+| frequency        | Frequency              | Yes      | Frequency of the forecasted time series                    |
+| start\_timestamp | string (ISO8601)       | Yes      | Start timestamp for the forecast horizon                   |
+| quantiles\*      | Dict\[str, TimeSeries] | No       | Map from quantile string (e.g., "0.1") to values           |
+| ...extra         | any                    | No       | Additional extensible fields                               |
+
+
+### Usage
+
+| Field              | Type | Required | Description                                 |
+| ------------------ | ---- | -------- | ------------------------------------------- |
+| prompt\_tokens     | int  | Yes      | Number of tokens in prompt (if LLM-related) |
+| completion\_tokens | int  | Yes      | Number of tokens in result                  |
+| total\_tokens      | int  | Yes      | Total tokens used                           |
+| ...extra           | any  | No       | Additional extensible fields                |
+
+
+### Error Handling
+
+If an error occurs, the response may contain a top-level `error` field, or errors may be reported per-forecast (in the corresponding output object). The structure is:
+
+```
+{
+  "error": {
+    "code": "<string>",
+    "message": "<description>",
+    "param": "<string>",
+    "type": "<string>"
+  }
+}
+```
+
+
+## Enumerations
+
+### Frequency
+
+* `SECOND`, `S`: second
+* `MINUTE`, `T`: minute
+* `HOUR`, `H`: hour
+* `DAY`, `D`: day
+* `WEEK`, `W`: week
+* `MONTH`, `M`: month
+* `QUARTER`, `Q`: quarter
+* `YEAR`, `Y`: year
+
+### Status
+
+* `COMPLETED`: The forecast completed successfully
+* `ERROR`: There was an error in processing
+* `PENDING`: The forecast is still running
+* `PARTIAL`: Partially completed
+
+### TimeSeriesType
+
+* `univariate_time_series`
+* `multivariate_time_series`
+
+
+## Examples
+
+### Start Service
+```
+python -m huggingfaceserver --model_id="google/timesfm-2.0-500m-pytorch" --model_name=timesfm2 --http_port=8090
```

Comment:
Isn't it better to provide a working example with KServe deployed on k8s?

Reply:
added, thx!